pkg/process/monitor/process_monitor.go

```diff
@@ -185,19 +222,20 @@ func (pm *ProcessMonitor) mainEventLoop() {
 				return
 			}
 
-			pm.eventCount.Inc()
+			pm.tel.events.Add(1)
+			pm.tel.eventsChannelSize.Set(int64(len(pm.netlinkEventsChannel)))
```

Comment:
can you show how the metric looks like?
I'm not sure if the measuring of the events channel makes sense

Reply:
it might be more relevant to set it every interval (like inside the logTicker case)

Reply:
the previous chan log was useless as it's mostly reported 0 or if it was a positive value it would be a very loaded environment.
I change the local counter to max chan size and report it every 2 minutes (ticker) only on log

Reply:
moving the discussion to https://github.com/DataDog/datadog-agent/pull/18018#discussion_r1274704913